helm-chart/images/binderhub/requirements.txt

```diff
@@ -1,7 +1,55 @@
-pycurl==7.43.0.1
-tornado==6.0.*
-kubernetes==9.0.*
-jupyterhub==1.1.0
-jsonschema==2.6.0
-# Logging sinks to send eventlogging events to
-google-cloud-logging==1.8.*
```

Comment:
google-cloud-logging is no longer within our image! I need to fix that.

Reply:
I think this is now resolved as the package appears in the output